### DISCUSSION TITLE
facing issue on Reference Many realtions.

### DISCUSSION THREAD

**Discussion by pksharma-ebiz** [loopbackio/loopback-next#9021] [Q&A]
Hi ,
I am facing issue when I am trying reference-many relation in my application where I want that a book can have multiple authors and I can save each author Id in an array.
I also setup emitDecoratorMetadata in tsc.config as true.
Can some guide me there

Cannot start the application. Error: Cannot infer type of model property Authors.prototype.id because TypeScript compiler option `emitDecoratorMetadata` is not set. Please enable `emitDecorato
rMetadata` or use the second argument of `@referencesMany` decorator to specify the property type explicitly.

**Comment by achrinza** [comment]
Hi @pksharma-ebiz, currently LoopBack 4 does not support References* relations, and `@referencesMany` is a placeholder for the feature. Hence why it's not documented on the LB4 Docs.

Consider using `@HasMany` instead, if needed.

I understand that this may not be answer you're looking for, but I do hope it helps provide insight onto the feature status.
